com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java

```diff
@@ -49,7 +46,6 @@
     public CompletableFuture<Response> handle(Command command, Arguments arguments, Response response, IDebugAdapterContext context) {
         AttachArguments attachArguments = (AttachArguments) arguments;
         context.setAttached(true);
-        context.setSourcePaths(attachArguments.sourcePaths);
```

Comment:
Why do we remove this line?

Reply:
Follow yaohai's comments, I don't remove it, but extract the common code of launchHandler and attachHandler to the new PostLaunchAndAttachHandler.